repos/spack_repo/builtin/packages/boost/package.py

```diff
@@ -317,6 +317,9 @@ def libs(self):
     # safe to do so on affected platforms.
     conflicts("+clanglibcpp", when="@1.85: +stacktrace")
 
+    # https://github.com/boostorg/python/issues/400
+    conflicts("@:1.80.0", when="+python ^python@3.11:", msg="Boost.python.enum has a known bug for older versions")
```

Comment:
A small clarification to the message.

```suggestion
    conflicts("@:1.80.0", when="+python ^python@3.11:", msg="Boost.python.enum has a known bug for python before 3.11")
```

Reply:
Still not clear. how about:
"Boost.python.enum has a known bug for python 3.11"

Reply:
It's any version of python newer than 3.11. What if we use the language of spack and just say

```suggestion
    conflicts("@:1.80.0", when="+python ^python@3.11:", msg="Boost.python.enum has a known bug for boost@:1.80.0 and python@3.11:")
```